### PR TITLE
Address review comments from merged PRs

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: Tests/SwiftParserTest/translated/InitDeinitTests.swift [hunk 1]
```diff
@@ -154,7 +156,7 @@ final class InitDeinitTests: XCTestCase {
         DiagnosticSpec(message: "deinitializers cannot have parameters", fixIts: ["remove function signature"]),
       ], fixedSource: """
       class FooClassDeinitializerA {
-        deinit{}
+        deinit {}
```
Comment (bnbarham) [inline Tests/SwiftParserTest/translated/InitDeinitTests.swift:159]: Nice :)

File: Tests/SwiftParserTest/translated/GuardTests.swift [hunk 2]
```diff
@@ -24,7 +24,7 @@ final class GuardTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected conditions in 'guard' statement"),
+        DiagnosticSpec(message: "expected condition in 'guard' statement"),
```
Comment (bnbarham) [inline Tests/SwiftParserTest/translated/GuardTests.swift:27]: Thanks!

**Review by bnbarham — APPROVED** [review]

**Comment by ahoppen** [comment]
@swift-ci Please test

**Pull request merged**